vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -551,14 +551,14 @@ - (void)saveBundleState:(std::unique_ptr<ads::BundleState>)state callback:(ads::
 
 #pragma mark - Logging
 
-- (void)eventLog:(const std::string &)json
+- (void)log:(const char *)file line:(const int)line severity:(const ads::LogSeverity)severity message:(const std::string &) message
 {
-  [self log:__FILE__ line:__LINE__ logLevel:ads::LogLevel::LOG_INFO]->stream() << json << std::endl;
+  [self log:__FILE__ line:__LINE__ severity:severity message:message];
```

Comment:
@kylehickinson iOS was logging all logs as `LOG_INFO`, was this expected as some verbose logging is quite intense?

Reply:
The method you are referring to (`eventLog`) used `LOG_INFO` because no severity was passed (you added that in this PR). You can see the original log method used severities

Reply:
So I did :-), I can only blame a long day...